NitroxPatcher/Patches/Dynamic/LiveMixin_AddHealth_Patch.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Harmony;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.GameLogic.Helper;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Core;
+using NitroxModel.DataStructures;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using NitroxModel_Subnautica.DataStructures;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    class LiveMixin_AddHealth_Patch : NitroxPatch, IDynamicPatch
```

Comment:
If possible, let's see if we can federate out some of the state keeping and logic into a new class that we invoke.

Reply:
You mean something like an LiveMixin manager? I hesitated because i was not sure if we will use that for other entities. If wanted, i will refactor this into it's own management class.